editor/src/clj/editor/analytics.clj

```diff
@@ -33,15 +33,15 @@
 ;; Set this to true to see the events that get sent when testing.
 (defonce ^:private log-events? false)
 
-(defonce ^:private batch-size 16)
+(defonce ^:private batch-size 25)
 (defonce ^:private cid-regex #"[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
 (defonce ^:private config-atom (atom nil))
 (defonce ^:private event-queue-atom (atom PersistentQueue/EMPTY))
 (defonce ^:private max-failed-send-attempts 5)
-(defonce ^:private payload-content-type "application/x-www-form-urlencoded; charset=UTF-8")
+(defonce ^:private payload-content-type "application/json")
 (defonce ^:private shutdown-timeout 1000)
-(defonce ^:private uid-regex #"[0-9A-F]{16}")
 (defonce ^:private worker-atom (atom nil))
+(defonce ^:private session-id (rand))
```

Comment:
I need something random for a session id while the editor is running

Reply:
Could have also used `(str (UUID/randomUUID))` that's used on some other places in this ns.